packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -562,6 +562,7 @@ void runPigeonIntegrationTests(TargetGenerator targetGenerator) {
         'a': 1,
         'b': 2.3,
         'c': 'four',
+        'd': null,
```

Comment:
is `d` here supposed to be an array type itself? 

Basically idea is we want the integration test to pass with the fix, and fail without the fix